Use _url instead of _path

mtsmfm opened the pull request:
context: https://github.com/bm-sms/daimon-news-multi-tenant/pull/206#discussion_r51962308

`git grep --name-only ' [a-z_]*_path' | grep -v test | xargs -L1 ruby -i -e 'puts gets(nil).to_s.gsub(/( [a-z_]*)_path(?!:)/, "\\1_url")'`

tricknotes commented:
Seems good :sunny:

okkez commented on app/views/posts/show.html.haml:
```diff
@@ -1,7 +1,7 @@
   - content_for(:rel_next_prev_link_tags) { rel_next_prev_link_tags(@pages) }
   - content_for(:canonical_tag) do
-    %link{rel: :canonical, href: url_for(all: true, only_path: false)}
+    %link{rel: :canonical, href: url_for(all: true, only_url: false)}
```
:eyes: Is this correct?

mtsmfm replied:
OMG

mtsmfm replied:
I'll add test for `only_path` option

okkez replied:
http://api.rubyonrails.org/classes/ActionDispatch/Routing/UrlFor.html#method-i-url_for

`only_path: false` is correct for this purpose.

mtsmfm committed:
Use _url instead of _path

context: https://github.com/bm-sms/daimon-news-multi-tenant/pull/206#discussion_r51962308

`git grep --name-only ' [a-z_]*_path' | grep -v test | xargs -L1 ruby -i -e 'puts gets(nil).to_s.gsub(/( [a-z_]*)_path(?!:)/, "\\1_url")'`

mtsmfm committed:
Add test for canonical

mtsmfm commented:
Added test

okkez commented:
LGTM.
